service/controller/resource/configmap/desired.go

```diff
@@ -63,6 +65,11 @@ func (r *Resource) newConfigMaps(customResource v1alpha1.KVMConfig) ([]*apiv1.Co
 		return nil, microerror.Mask(err)
 	}
 
+	defaultVersions := key.DefaultVersions()
```

Comment:
My 2c:
I don't like these defaultVersion things - Why not just create the struct and set all values?

Reply:
I thought this was a placeholder until ignition-operator took over the rendering. I'm not a huge fan of the implementation here but I do like the idea of keeping some sane defaults in one place as opposed to scattered across the different resources